src/libs/SidebarUtils.js

```diff
@@ -263,7 +259,7 @@ function getOptionData(reportID) {
     let lastMessageText = hasMultipleParticipants && lastActorDetails && (lastActorDetails.login !== currentUserLogin.email)
         ? `${lastActorDetails.displayName}: `
         : '';
-    lastMessageText += report ? lastMessageTextFromReport : '';
```

Comment:
Is there a case when the report is falsey? if yes, would the ` ReportUtils.getLastMessageText(report);` lead to an error?